tests/management/commands/test_print_settings.py

```diff
@@ -34,6 +35,16 @@ def test_with_multiple_setting_args(capsys):
     assert 'DATABASES' in out
     assert 'INSTALLED_APPS' in out
 
+def test_with_not_show_secrets(capsys):
+    call_command(
+        'print_settings',
+        'SECRET_KEY',
+    )
+
+    out, err = capsys.readouterr()
+
+    assert '*******' in out
```

Comment:
Can we make the check a little bit more explicit ?

eg use `--format=pprint` that would allow checking for `{'SECRET_KEY': 'dummy'}`

```
$ ./manage.py print_settings --format=pprint SECRET_KEY
{'SECRET_KEY': 'dummy'}
```

Reply:
and adding an explicit check for the variant with `--show-secrets` would be even cooler ;-)

Reply:
Sure!